dkc/core/rest/folder.py

```diff
@@ -94,6 +94,12 @@ class Meta(FolderSerializer.Meta):
         read_only_fields = FolderSerializer.Meta.read_only_fields + ['parent']
 
 
+class FolderQuotaSerializer(serializers.ModelSerializer):
```

Comment:
```suggestion
class QuotaSerializer(serializers.ModelSerializer):
```

Internally, `Quota` isn't really specific to a `Folder`.